src/genie/libs/parser/iosxr/show_interface.py

```diff
@@ -3032,4 +3032,72 @@ def cli(self, interface=None, output=None):
         else:
             out = output
 
-        return super().cli(output=out)
\ No newline at end of file
+        return super().cli(output=out)
+
+
+#############################################################################
+# Parser For show interface summary
+#############################################################################
+
+class ShowInterfaceSummarySchema(MetaParser):
+    """schema for show interface summary
+    """
+
+    schema = {
+        'interface_types': {
+            Any(): {
+                'total': str,
+                'up': str,
+                'down': str,
+                'admin_down': str,
+            }
+        }
+    }
+
+
+class ShowInterfaceSummary(ShowInterfaceSummarySchema):
+    """parser for show interface summary
+    """
+
+    cli_command = ['show interface summary']
```

Comment:
Is this meant to be?

```suggestion
    cli_command = ['show interface summary', 'show interface {interface} summary']
```

Reply:
Doesn't ''show interface {interface} summary'' produce the same output as "show interface summary"? I am not quite sure why the second command is needed? If so, I sure can handle both.

Reply:
Ah nevermind, this was just a plain copy&paste error from my side. I forgot to remove the logic for the multiple command part. Will fix this to only support "show interface summary"